src/main/java/fr/mrc/ptichat/connection/RequestRunnable.java

```diff
@@ -8,29 +10,45 @@
  */
 public class RequestRunnable implements Runnable {
 
-    private boolean resume = false;
     private BufferedReader in;
     private InetAddress address;
     private int port;
-    private String message = null;
+    private RequestHandler rh = new RequestHandler();
+    private Flag flag;
 
-    public RequestRunnable(BufferedReader in, InetAddress address, int port){
+    public RequestRunnable(BufferedReader in, InetAddress address, int port, Flag flag){
```

Comment:
Point out in comments the fact that `Flag flag` is reusing the same flag as Response Runnable ?